webrtc.html

```diff
@@ -1932,26 +1935,29 @@
                   set and the set of certificates differs from the ones used
                   when <var>connection</var> was constructed, throw an
                   <code>InvalidModificationError</code>.</li>
```

Comment:
nit: This paragraph regarding certificates could match the format of the new sections you added with InvalidModificationErrors